include/libbpf/bpf_endian.h

```diff
@@ -0,0 +1,99 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
Please copy the license files listed below to `include/libbpf`:

- https://github.com/libbpf/libbpf/blob/master/LICENSE
- https://github.com/libbpf/libbpf/blob/master/LICENSE.BSD-2-Clause
- https://github.com/libbpf/libbpf/blob/master/LICENSE.LGPL-2.1